pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -926,3 +940,18 @@ func (r *Builder) GetPopulatorTaskName(pvc *core.PersistentVolumeClaim) (taskNam
 	err = planbase.VolumePopulatorNotSupportedError
 	return
 }
+
+func getDeviceNumber(deviceString string) int {
+	if !strings.HasPrefix(deviceString, "/dev/sd") || len(deviceString) < 8 {
```

Comment:
i really don't like the magic numbers here.
how about making a variable such as:
`diskPrefix := "/dev/sd"`
and then use it's length? 
`if !strings.HasPrefix(deviceString, diskPrefix) || len(deviceString) <= len(diskPrefix) {`?